src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -423,6 +423,12 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('sync_schedule', options_list=['--sync-schedule', '-s'], help='Optional parameter to define the sync schedule. Uses cron expression to determine the schedule. If not specified, the instance is considered always online and attempts to sync every minute.', required=False)
         c.argument('sync_message_ttl', help='Determines how long the sync messages will be kept in the cloud. Uses ISO 8601 duration format.', required=False)
 
+    with self.argument_context('acr connected-registry repo') as c:
+        c.argument('add_repos', options_list=['--add'], nargs='*', required=False,
+                   help='repository permissions to be added to the targeted connected registry and it\'s ancestors sync scope maps. Use the format "--add [REPO1 REPO2 ...]" per flag. ' + repo_valid_actions)
+        c.argument('remove_repos', options_list=['--remove'], nargs='*', required=False,
+                   help='respsitory permissions to be removed from the targeted connected registry and it\'s succesors sync scope maps. Use the format "--remove [REPO1 REPO2 ...]" per flag. ' + repo_valid_actions)
+
```

Comment:
`Commands must follow a "[noun] [noun] [verb]" pattern` (See the first item in [command_guideline#command-naming-and-behavior](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#command-naming-and-behavior)). You may consider using 2 separate commands: `acr connected-registry repo add` and `acr connected-registry repo remove` with `--repos` argument.

Reply:
could I use --add-repos --remove-repos ?

Reply:
I would like to keep them in the same command if possible

Reply:
I see, you may use `--add` and `--remove` together in one command.